src/VisualStudio/Core/Impl/Options/GridOptionPreviewControl.xaml

```diff
@@ -35,12 +35,18 @@
             <converters:MarginConverter x:Key="MarginConverter" />
         </Grid.Resources>
         <Grid.RowDefinitions>
+            <RowDefinition Height="30" />
             <RowDefinition Height="*" />
             <RowDefinition Height="5" />
             <RowDefinition Height="*" />
         </Grid.RowDefinitions>
+        <TextBlock Text="{x:Static local:GridOptionPreviewControl.CodeStylePageHeader}" >
+            <Hyperlink RequestNavigate="LearnMoreHyperlink_RequestNavigate" NavigateUri="{x:Static local:GridOptionPreviewControl.CodeStylePageHeaderLearnMoreUri}">
```

Comment:
:bulb: You should be able to use `Margin="2,0"` on either the Hyperlink or the nested TextBlock element to add the space.

Reply:
Added <Run Text=" "/> after the CodeStylePageHeader text
Hyperlink does not support Margin; if I add Margin to TextBlock, then I can see the hyperlink underline actually starts with an empty space
![image](https://user-images.githubusercontent.com/17056651/35075434-d1b1f82e-fba7-11e7-97a2-9a35b3886ae4.png)